save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestService.kt

```diff
@@ -59,7 +60,7 @@ class TestService(
                     val testSuiteStub = TestSuite(testRootPath = "N/A").apply {
                         id = testDto.testSuiteId
                     }
-                    Test(testDto.hash, testDto.filePath, testDto.pluginName, LocalDateTime.now(), testSuiteStub, testDto.tags.joinToString(";"))
+                    Test(testDto.hash, FilenameUtils.separatorsToUnix(testDto.filePath), testDto.pluginName, LocalDateTime.now(), testSuiteStub, testDto.tags.joinToString(";"))
```

Comment:
what's about existed values in database? maybe need to ad liquibase migration script?